osbs/build/user_params.py

```diff
@@ -80,10 +85,17 @@ def set_params(self,
                    flatpak=None, reactor_config_map=None, reactor_config_override=None,
                    yum_repourls=None, signing_intent=None, compose_ids=None,
                    isolated=None, scratch=None, parent_images_digests=None,
+                   tags_from_yaml=None, additional_tags=None, container_yaml=None,
+                   git_commit_depth=-1,
                    **kwargs):
         self.git_uri.value = git_uri
         self.git_ref.value = git_ref
         self.git_branch.value = git_branch
+        self.git_commit_depth.value = git_commit_depth
+        self.tags_from_yaml.value = tags_from_yaml
+        self.additional_tags.value = additional_tags or {}
```

Comment:
did you mean `or set()` IIRC additional_tags var has type set.

Reply:
Fixed, thanks.